assets/js/common/SuseManagerSettingsDialog/SuseManagerSettingsModal.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { noop } from 'lodash';
+import { format } from 'date-fns';
+import { EOS_LOCK_OUTLINED } from 'eos-icons-react';
+
+import Button from '@common/Button';
+import Modal from '@common/Modal';
+import Input, { Password, Textarea } from '@common/Input';
+import Label from '@common/Label';
+
+import { hasError } from './errors';
+
+const defaultErrors = [];
```

Comment:
Does this have a purpose?

Reply:
It does! If you specify empty arrays, empty objects or functions (mainly the first two) as default props without declaring them as `const` you incur into their reference constantly changing when rendering and components just get a new prop ref every time they render, ending up in endless rerendering. Your browser gets easily stuck on those, crashing and looping.

Reply:
More info here https://github.com/facebook/react/issues/18123

Reply:
ahhh very tricky! Good to know :brain: